cli/cmd/edges_test.go

```diff
@@ -0,0 +1,152 @@
+package cmd
+
+import (
+	"testing"
+
+	"github.com/linkerd/linkerd2/controller/api/public"
+)
+
+type edgesParamsExp struct {
+	options      *edgesOptions
+	resSrc       []string
+	resDst       []string
+	resClient    []string
+	resServer    []string
+	resMsg       []string
+	resourceType string
+	file         string
+}
+
+var (
```

Comment:
AFAICT, these variables are only used in `TestEdges(t *testing.T)`, right? I'd prefer to make them local variables inside the test function. Otherwise, these global variables are accessible by everything within the `cmd` package.

Reply:
yup, these are only local variables. will move them inside.